definitions/ext-vpc_network/aws-kinesis-firehose-vpc-flow-logs-dashboard.json

```diff
@@ -1,167 +1,463 @@
+            "height": 3,
+            "width": 2
           },
-          "title": "",
+          "title": "VPC Network Summary",
           "rawConfiguration": {
-            "text": "# VPC Flow Log Analytics\n---\n\n![](https://d0.awsstatic.com/logos/powered-by-aws.png)\n\n\n\n"
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Log_VPC_Flows SELECT latest(vpc_id) AS 'VPC Name', latest(instrumentation.provider) AS 'Flow Type', rate(count(*),1 second) as 'Flows per Second', latest(timestamp) as 'Last Update', uniqueCount(srcaddr, dstaddr) as 'Observed IP Addresses', uniqueCount(src_endpoint, dest_endpoint) as 'Observed Endpoint Tuples'"
+              }
+            ],
+            "thresholds": []
           }
         },
         {
           "visualization": {
-            "id": "viz.table"
+            "id": "viz.area"
           },
           "layout": {
-            "column": 4,
+            "column": 3,
             "row": 1,
             "height": 3,
-            "width": 9
+            "width": 5
           },
-          "title": "Detected Anomalies",
+          "title": "Flow Bytes",
           "rawConfiguration": {
             "dataFormatters": [],
             "facet": {
               "showOtherSeries": false
             },
+            "legend": {
+              "enabled": true
+            },
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "FROM WatcherSignalDeviation SELECT facetAttribute AS 'Type', facetValue AS 'IP', status, title AS 'Anomaly', comparisonWindowStandardDeviation AS 'Z-Score', lookoutLink AS 'Explore via Lookout' WHERE status IN('STARTED','ONGOING','RESOLVED')"
+                "query": "FROM Log_VPC_Flows\nSELECT sum(bytes*numeric(sample_rate)) \nFacet CASES (\n  WHERE flow_direction = 'egress' as 'Egress Bytes',\n  WHERE flow_direction = 'ingress' as 'Ingress Bytes') \n TIMESERIES 5 MINUTES"
               }
-            ]
+            ],
+            "yAxisLeft": {
+              "zero": true
+            }
           }
         },
         {
           "visualization": {
-            "id": "viz.billboard"
+            "id": "viz.area"
+          },
+          "layout": {
+            "column": 8,
+            "row": 1,
+            "height": 3,
+            "width": 5
+          },
+          "title": "Flow Packets",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Log_VPC_Flows\nSELECT sum(packets*numeric(sample_rate))\nFacet CASES (\n  WHERE flow_direction = 'egress' as 'Egress Packets',\n  WHERE flow_direction = 'ingress' as 'Ingress Packets') \n TIMESERIES 5 MINUTES"
```

Comment:
similar to above; unlike traditional flow; VPC flows apply their sample rate to the flow records, not the packets; this math doesn't work